### ISSUE TITLE
Proj0805 gives false positive with Central Package Management and Common.props file

### ISSUE THREAD

**Issue by Gemberkoekje** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#159]
A csproj file like:
```
<Project Sdk="Microsoft.NET.Sdk">

  <Import Project="../../props/common.props" />
```
A common.props file like:
```
<?xml version="1.0" encoding="utf-8"?>
<Project>
  <ItemGroup Label="Analyzers">
    <PackageReference Include="AsyncFixer" PrivateAssets="all" />
  </ItemGroup>
</Project>

```
And a Directory.Packages.Props file like:
```
<Project>
  <PropertyGroup>
    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
    <CentralPackageFloatingVersionsEnabled>true</CentralPackageFloatingVersionsEnabled>
  </PropertyGroup>
  <ItemGroup>
    <PackageVersion Include="AsyncFixer" Version="*-*" />
  </ItemGroup>
</Project>

```

Returns the warning:
common.props: warning Proj0805: Define version for 'AsyncFixer' PackageReference. (https://dotnet-project-file-analyzers.github.io/rules/Proj0805.html)
